(Dashboards) Update List Widget supported datasets (#37746)
DataDog/documentation: content/en/dashboards/widgets/list.md

### Configuration

1. Choose the type of data to graph. The list widget supports many data sources, depending on which products are enabled for your organization. For the full list, see [Supported data sources](#supported-data-sources).

2. Set display preferences. On screenboards and notebooks, choose whether your widget has a custom timeframe or uses the global timeframe.

3. Optional: Give your graph a title (or leave blank for suggested title).

### Supported data sources

> The data sources available in the source dropdown depend on which products are enabled for your organization. The following table lists each data source, the data it displays, and any product requirements. Click a data source in the table to navigate to its configuration options.

Reviewer: ```suggestion
The data sources available in the source dropdown depend on which products are enabled for your organization. The following table lists each data source, the data it displays, and any product requirements. Where available, click a data source in the table to navigate to its configuration options.
```

Data sources marked _(Preview)_ are in Preview and might not be available in your organization.

| Data source | Description | Requirements |
|-------------|-------------|--------------|
| [Audit Trail](#options) | Audit events that track activity across your organization. | Audit Trail |
| [Cases](#cases) | Cases that track and triage work across teams. | Case Management |
| [CD Deployments](#options) _(Preview)_ | Continuous delivery deployment executions. | CD Visibility |
| [CI Pipeline](#ci-pipeline) | CI pipeline executions. | CI Pipeline Visibility |
| [CI Test](#options) | CI test runs. | Test Optimization |
| [Data Observability](#data-observability-preview) _(Preview)_ | Data assets, such as datasets and lineage, from Data Observability. | Data Observability |
| [Database Recommendations](#database-recommendations-preview) _(Preview)_ | Optimization recommendations from Database Monitoring. | Database Monitoring |

> | [DDSQL Editor](#notebook-ddsql-editor-reference-tables-and-developer-portal) | Results of a DDSQL query. | - |

Reviewer: What does the dash mean? Is it worth clarifying as "No requirements" or "None"?

| [Detection Rules](#detection-rules-preview) _(Preview)_ | Security detection rules. | Cloud SIEM or Cloud Security |

> | [Developer Portal](#notebook-ddsql-editor-reference-tables-and-developer-portal) _(Preview)_ | Views of software entities, including services, APIs, and datastores. | Internal Developer Portal |

Reviewer: ```suggestion
| [Developer Portal](#notebook-ddsql-editor-reference-tables-and-developer-portal) _(Preview)_ | Views of software entities, including services, APIs, and data stores. | Internal Developer Portal |
```

| [Security Signals](#options) _(Preview)_ | Security signals generated by detection rules. | Cloud SIEM |
| [Spans](#spans-and-watchdog-alerts) | APM spans. | APM |
| [Watchdog Alerts](#spans-and-watchdog-alerts) | Alerts surfaced by Watchdog. | - |
| [Workload Protection Agent](#workload-protection-agent-preview) _(Preview)_ | Workload Protection events from the Datadog Agent. | Workload Protection |

**Note:** The **Recommendations** data source displays Cloud Cost Management recommendations only. APM recommendations are not available as a List widget data source. If Cloud Cost Management is not configured for your organization, the widget displays a `Not Accessible` message. This indicates that the data source requires Cloud Cost Management, rather than that you lack permissions.

### Options

Each data source has its own configuration. For most data sources, you can:

- Select which **columns** to display.
- **Sort** the list by choosing a column and a direction (ascending or descending). The available sort columns are the columns displayed in the widget.
- Narrow results with a **search query**.

The following data sources have additional or different options.

> #### Cases

Reviewer: Consider making these data sources collapsible.

#### Logs

Group by:

- Patterns
- Transactions

Depending on your logs configuration, you can also select a storage location: Standard Indexes, Standard Indexes + Flex Logs, or Online Archives.

For the message column, you can choose how many lines to display (1, 3, or 10).

#### Notebook, DDSQL Editor, Reference Tables, and Developer Portal

These data sources display rows from a saved dataset or table:

- **Notebook** and **DDSQL Editor**: select a published dataset.
- **Reference Tables**: select a reference table.
- **Developer Portal**: select a software entity table, such as services, APIs, or datastores.

For these data sources, you can:

- Set **Display first** to limit the number of rows (10, 25, 50, 100, 500, or 1000, or a custom value).
- Toggle **Show All Columns**, or select up to 12 columns to display.
- Sort by clicking a column's sort icon.
- Filter the rows with a search query.

#### On-Call

Select a **Team**, and optionally add **Tags** to filter the events shown.

#### Recommendations

The columns for the Recommendations data source are fixed and cannot be customized.

> #### Spans and Watchdog alerts

Reviewer: Is this the product Watchdog Alerts? Or the alerts from Watchdog Alerts?

```suggestion
#### Spans and Watchdog Alerts
```
